File: Java/Car.java
class Car
{
    private Integer id;
    private String license;
    private Account driver;
    protected Integer passenger;

    public Car(String license, Account driver)
    {
        this.license = license;
        this.driver = driver;
    }

    void printDataCar()
    {
        if(passenger != null)
        {
            System.out.println("Car Lisence: " + license + " Driver: " + driver.name + " Passengers: " + passenger);
        }
    }

    public Integer getPassenger()
    {
        return passenger;
    }

    public void setPassenger(Integer passenger)
    {
        if(passenger == 4)
        {
            this.passenger = passenger;
        }
        else
        {
            System.out.println("You need to assign 4 passengers");
        }
    }

    public Integer getId()
    {
        return id;
    }

    public void setId(Integer id)
    {
        this.id = id;
    }

    public String getLicense()
    {
        return license;
    }

    public void setLicense(String license)
    {
        this.license = license;
    }

    public Account getDriver()
    {
        return driver;
    }

    public void setDriver(Account driver)
    {
        this.driver = driver;
    }

    
}

File: PHP/Car.php
<?php
require_once("Account.php");

class Car
{
    public $d;
    public $license;
    public $driver;
    public $passenger;
    
    public function _constuct($license, $driver)
    {
        $this->licence = $license;
        $this->driver = $driver
    }
    
    public function printDataCar()
    {
        echo "License: ".$this->license."<br>";
        echo "Name: ".$this->driver->name."<br>";
        echo "Document: ".$this->driver->document."<br>";
    }
}

?>